Application/EdFi.Ods.Common/Models/Domain/PropertyType.cs

```diff
@@ -12,7 +13,7 @@ namespace EdFi.Ods.Common.Models.Domain
 {
     public class PropertyType
     {
-        public PropertyType(DbType dbType, int maxLength = 0, int precision = 0, int scale = 0, bool isNullable = false)
+        public PropertyType(DbType dbType, int maxLength = 0, int precision = 0, int scale = 0, bool isNullable = false, decimal? minValue = null, decimal? maxValue = null)
```

Comment:
No change required here, but just adding a note for posterity.

If we ever need to support min/max values for anything other than integers or decimal values (e.g. dates), we'll probably want to move the min/max elements out of the PropertyType structure (into a new PropertyConstraints structure, along with the isNullable value?) which has now grown beyond its originally intended scope.